webui/react/src/utils/fonts.ts

```diff
@@ -0,0 +1,43 @@
+export const ThemeFont = {
+  Code: 'var(--theme-font-family-code)',
+  UI: 'var(--theme-font-family)',
+};
```

Comment:
You need an `as const` on these so they infer correctly, you should also export a matching type. See other cases where we use enums as an example.